paywall/src/components/LandingPage.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import styled from 'styled-components'
+import Layout from './interface/Layout'
+import KeyText from './interface/svg/KeyText'
+import Media from '../theme/media'
+
+export default function LandingPage() {
+  return (
+    <Layout title="Paywall" forContent>
+      <section>
+        <Title>Pay for Content Seamlessly</Title>
+        <SubTitle>
+          Unlock enables anyone with a crypto wallet to seamlessly buy and
+          manage access to any kind of content.
+        </SubTitle>
+      </section>
+      <Content>
+        <KeyGraphic height="245px" width="200px" />
+        <p>
+          At Unlock, we believe that the more accessible paid content is the
+          better it will be. To do that we’re making it easy for readers like
+          you to seamlessly pay for and manage your content.
+        </p>
+        <p>
+          If you want to know more about Unlock’s decentralized payment protocol
+          check out our blog.
+        </p>
+      </Content>
+      <section>
+        <Divider>FAQ</Divider>
+        <Question>How do I get a wallet?</Question>
+        <Answer>
+          You need a web3 enabled wallet such as MetaMask for Chrome or Opera
+          for Android.
+        </Answer>
+        <Question>What does it cost to use Unlock?</Question>
+        <Answer>
+          Unlock is free to use though you will have to pay a small transaction
+          fee when purchasing your content. It’s usually just a few cents on top
+          of your purchase.
+        </Answer>
+        <Question>What about privacy?</Question>
+        <Answer>
+          Unlock is built on top of the Ethereum blockchain and therefore offers
+          a number of great privacy and security features....
```

Comment:
I think rather than an ellipsis here, those great privacy and security features should be listed!

Reply:
I happen to agree. @julien51 and @smombartz it would be useful to understand the future path of this portion of the landing page?

Reply:
@benwerd too, actually!